### ISSUE TITLE
Post install GeoIP update

### ISSUE THREAD

**Issue by kraigu** [ncsa/bro-cluster-in-a-box-setup#1]
Post install:
Received Error -21 (Sanity check database_info string failed) when attempting to update GeoIP Database

Probably Maxmind changes?

**Comment by JustinAzoff** [comment]
yeah.. I know the DB we had been using for years was the "Legacy" one, so it might have finally been discontinued.
